iothub/service/tests/ScheduledJobsClientTests.cs

```diff
@@ -0,0 +1,484 @@
+            var mockHttpRequestFactory = new HttpRequestMessageFactory(s_httpUri, "");
+
+            using var mockHttpResponse = new HttpResponseMessage
+            {
+                StatusCode = HttpStatusCode.OK,
+                Content = HttpMessageHelper.SerializePayload(scheduledJob),
+            };
+            mockHttpResponse.Headers.Add("ETag", "\"AAAAAAAAAAE=\"");
+
+            var mockHttpClient = new Mock<HttpClient>();
+            mockHttpClient
+                .Setup(restOp => restOp.SendAsync(It.IsAny<HttpRequestMessage>(), It.IsAny<CancellationToken>()))
+                .ReturnsAsync(mockHttpResponse);
+
+            var qureyClient = new QueryClient(
+                HostName,
+                mockCredentialProvider.Object,
+                mockHttpClient.Object,
+                mockHttpRequestFactory,
+                s_retryHandler);
+
+            var scheduledJobsClient = new ScheduledJobsClient(
+                HostName,
+                mockCredentialProvider.Object,
+                mockHttpClient.Object,
+                mockHttpRequestFactory,
+                qureyClient,
+                s_retryHandler);
+
+            // act
+            Func<Task> act = async () => await scheduledJobsClient.GetAsync("foo");
+
+            // assert
+            await act.Should().NotThrowAsync();
+        }
+
+        [TestMethod]
+        public async Task ScheduledJobsClient_GetAsync_NullArgumentThrows()
+        {
+            // arrange
+            var scheduledJob = new ScheduledJob();
+            var mockCredentialProvider = new Mock<IotHubConnectionProperties>();
+            mockCredentialProvider
+                .Setup(getCredential => getCredential.GetAuthorizationHeader())
+                .Returns(s_validMockAuthenticationHeaderValue);
+            var mockHttpRequestFactory = new HttpRequestMessageFactory(s_httpUri, "");
+
+            using var mockHttpResponse = new HttpResponseMessage
+            {
+                StatusCode = HttpStatusCode.OK,
+                Content = HttpMessageHelper.SerializePayload(scheduledJob),
+            };
+            mockHttpResponse.Headers.Add("ETag", "\"AAAAAAAAAAE=\"");
+
+            var mockHttpClient = new Mock<HttpClient>();
+            mockHttpClient
+                .Setup(restOp => restOp.SendAsync(It.IsAny<HttpRequestMessage>(), It.IsAny<CancellationToken>()))
+                .ReturnsAsync(mockHttpResponse);
+
+            var qureyClient = new QueryClient(
+                HostName,
+                mockCredentialProvider.Object,
+                mockHttpClient.Object,
+                mockHttpRequestFactory,
+                s_retryHandler);
+
+            var scheduledJobsClient = new ScheduledJobsClient(
+                HostName,
+                mockCredentialProvider.Object,
+                mockHttpClient.Object,
+                mockHttpRequestFactory,
+                qureyClient,
+                s_retryHandler);
+
```

Comment:
For this test, you may want to refer to this test I wrote (https://github.com/Azure/azure-iot-sdk-csharp/blob/choisophia/improve-code-coverage-devices/iothub/service/tests/Messaging/MessageClientTests.cs#L115). I think these lines are not necessary.